docs/quick_start/for_artists.rst

```diff
@@ -3,4 +3,24 @@
   Copyright Contributors to the OpenColorIO Project.
 
 Quick Start for Artists
-=======================
\ No newline at end of file
+=======================
+
+Most users will likely want to use the OpenColorIO that comes precompiled with
+their applications.  See the :ref:`compatiblesoftware` for further details on
+each application.
+
+Note that OCIO configurations are required to do any 'real' work, and are
+available separately on the :ref:`downloads` section of this site. Example
+images are also available. For assistance customizing .ocio configurations,
+contact `ocio-user <https://lists.aswf.io/g/ocio-user>`__\.
+
+- Step 1:  set the OCIO environment-variable to /path/to/your/profile.ocio
+- Step 2:  Launch supported application.
+
+If you are on a platform that is not envvar friendly, most applications also
+provide a menu option to select a different OCIO configuration after launch.
+
+Please be sure to select a profile that matches your color workflow (VFX work
```

Comment:
Recommend using "config" or "configuration" instead of "profile" to avoid confusion with ICC profiles.

Reply:
Changed.  It seems Jeremy started with "profile" and changed to "configuration" at some point?  Would be interesting to know the history.